bigbluebutton-html5/imports/ui/components/error-screen/component.jsx

```diff
@@ -6,19 +6,15 @@ import styles from './styles.scss';
 const intlMessages = defineMessages({
```

Comment:
Removing all those defaultMessages in the Error page will break the client if a request to load the localization fails for example. And not only that, having only an id to understand what a message means is pretty hard. I dont understand why you guys are removing all defaultMessages and descriptions from the intl messages.